### DISCUSSION TITLE
How to disable commit message preview while hovering the commit history?

### DISCUSSION THREAD

**Discussion by ibrahimdanish** [gitextensions/gitextensions#10155] [Q&A]
**Do you want to request a *feature* or report a *bug*?**
Feature
**What is the current behavior?**
Unable to disable the commit message preview when we just hover the repo commit hostory as it is quite annoying.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
N/A
**What is the expected behavior?**
To be able to disable the commit message preview.
**Environment you encounter the issue:**

- Git Extensions 3.00.00.4433
- Build fca7cf228b481ee8c1b779cf7b882ccdfbdcd1bc
- Git 2.20.0.windows.1
- Microsoft Windows NT 10.0.16299.0
- .NET Framework 4.7.2650.0
- DPI 96dpi (no scaling)

**Did this work in previous version of GitExtensions (which)?**
Previous version did not have this preview commit message feature.

![commitpreview](https://user-images.githubusercontent.com/2067961/50009401-39494000-ffb7-11e8-9bb0-4d9d53e8aacb.png)

**Comment by gerhardol** [comment]
Can you please fill in the template and provide a screenshot?

**Comment by Al-Elisey** [comment]
Hello, join the question

**Comment by RussKie** [comment]
It's not possible.

https://github.com/gitextensions/gitextensions/blob/bb9b9673f4c2aa9e96410a2207712cced751f022/GitUI/UserControls/RevisionGrid/RevisionGridControl.cs#L247-L249

**Comment by vbjay** [comment]
Adding a context menu in view menu saying show commit tooltip and adjusting the provider to not show tooltips when turned off should work. If not that, can adjust provider to just always generate empty string when setting is off.
